.github/workflows/coverage.yml

```diff
@@ -45,4 +45,51 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: code-coverage-report
-          path: ./ff/serving/cover.html
\ No newline at end of file
+          path: ./ff/serving/cover.html
+
+  golangci:
+    name: Go Linter
+    defaults:
+      run:
+        working-directory: ./ff/serving
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+
+      - name: Install grpc_tools
+        run: pip install grpcio-tools
+
+      - name: Install Protobuf
+        run: sudo snap install protobuf --classic
+
+      - name: Setup Proto
+        run: ./gen_grpc.sh
```

Comment:
Why?

Reply:
Needed for the linter. Has import errors without it

Reply:
Ah I see, could you just add that in a comment just so its clear

Reply:
Done